### PR TITLE
add s3 buckets to formbuilder filestore

### PR DESCRIPTION
**Pull request by asmega**

### CODE REVIEW

**Comment by razvan-moj** [comment]
^ tiny edit `terraform fmt` please

**Comment by asmega** [comment]
sorry just noticed the the hiding whitespace

File: namespaces/cloud-platform-live-0.k8s.integration.dsd.io/formbuilder-platform-test-dev/resources/user-filestore.tf [hunk 1]
```diff
@@ -0,0 +1,29 @@
+# auto-generated from fb-cloud-platforms-environments
+##################################################
+# User Filestore S3
+module "user-filestore-s3-bucket" {
+  source = "github.com/ministryofjustice/cloud-platform-terraform-s3-bucket"
```
Comment (razvan-moj) [inline namespaces/cloud-platform-live-0.k8s.integration.dsd.io/formbuilder-platform-test-dev/resources/user-filestore.tf]: without a fixed reference like https://github.com/ministryofjustice/cloud-platform-terraform-s3-bucket/blob/19ed7266742ca67f095ecfa585fd8a0e80d63542/example/main.tf#L16 a future release might delete and recreate the bucket

Comment (asmega) [reply namespaces/cloud-platform-live-0.k8s.integration.dsd.io/formbuilder-platform-test-dev/resources/user-filestore.tf]: I see. Sorry, I'm new to terraform. is that assumed best practice? I just copied the the example from the README here https://github.com/ministryofjustice/cloud-platform-terraform-s3-bucket

Would you like me to update the README there too?

Comment (razvan-moj) [reply namespaces/cloud-platform-live-0.k8s.integration.dsd.io/formbuilder-platform-test-dev/resources/user-filestore.tf]: argh, we have an example in /example/ and another in the README sorry will correct

Comment (asmega) [reply namespaces/cloud-platform-live-0.k8s.integration.dsd.io/formbuilder-platform-test-dev/resources/user-filestore.tf]: I didn't spot that, I'll leave that in your capable hands 😄

**Pull request merged**